diff --git a/src/CMakeLists.txt b/src/CMakeLists.txt
index f8aeadc..3b5e7f8 100644
--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -65,7 +65,7 @@ set_target_properties(sms_static PROPERTIES OUTPUT_NAME sms)
 
 # targets must be added *after* all calls to link_libraries()
 add_executable(smsg "main.cpp")
-target_link_libraries(smsg PRIVATE sms_static)
+target_link_libraries(smsg PRIVATE sms_static gmp gmpxx)
 
 install(TARGETS
 	smsg
